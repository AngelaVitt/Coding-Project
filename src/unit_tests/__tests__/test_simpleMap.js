import GoogleMapReact from 'google-map-react';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import renderer from 'react-test-renderer';

export default class SimpleMap extends React.Component {
  

  

  render() {
    return (
      <GoogleMapReact
        defaultCenter={this.props.center}
        defaultZoom={this.props.zoom}
      />
    );
  }
}


test("renders correctly", () => {
  const renderer = new ShallowRenderer();

  const result = renderer.render(<SimpleMap/>);

  expect(result).toMatchSnapshot();
});
