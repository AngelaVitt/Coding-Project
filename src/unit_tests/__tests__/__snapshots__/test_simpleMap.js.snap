// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<GoogleMap
  debounced={true}
  defaultCenter={undefined}
  defaultZoom={undefined}
  distanceToMouse={[Function]}
  googleMapLoader={[Function]}
  heatmap={Object {}}
  heatmapLibrary={false}
  hoverDistance={30}
  layerTypes={Array []}
  options={[Function]}
  style={
    Object {
      "height": "100%",
      "margin": 0,
      "padding": 0,
      "position": "relative",
      "width": "100%",
    }
  }
  yesIWantToUseGoogleMapApiInternals={false}
/>
`;
